includes/class-lookup.php

```diff
@@ -103,6 +105,46 @@ public static function get_preservable_querystring_params_from_url( $url ) {
 		return array_intersect_key( $url_params, array_flip( $preservable_param_keys ) );
 	}
 
+		/**
+		 * Get redirect data status and url based on the provided url.
+		 *
+		 * To make the redirection match, we take a full URL as $url parameter, decode it and keep
+		 * only the PATH and QUERY part of it to look for known matches.
+		 *
+		 * @param string $url URL to find redirection for, can be a full size url,
```

Comment:
What's a "full size" URL?

Also, capitalize abbreviations:

```suggestion
		 * @param string $url URL to find redirection for, can be a full size URL,
```